lms/djangoapps/teams/csv.py

```diff
@@ -20,12 +21,76 @@ def load_team_membership_csv(course, response):
         response (HttpResponse): Django response object to which
             the CSV content will be written.
     """
-    # This function needs to be implemented (TODO MST-31).
-    _ = course
-    not_implemented_message = (
-        "Team membership CSV download is not yet implemented."
-    )
-    response.write(not_implemented_message + "\n")
+    team_membership_data = _lookup_team_membership_data(course)
+    headers = get_team_membership_csv_headers(course)
+    response.write(','.join(headers) + '\n')
+    for user_data in team_membership_data:
+        row = [user_data.get(header, '') for header in headers]
+        response.write(','.join(row) + '\n')
+
+
+def get_team_membership_csv_headers(course):
```

Comment:
Would it make sense to make this method private?

Reply:
yeah, may as well